src/app/foresight/integrations/page.tsx

```
@@ -0,0 +1,23 @@
+import { Button } from "@/components/ui/button";
+import { Label } from "@/components/ui/label";
+import { SOCIAL_PLATFORMS } from "@/constants/social-media";
+
+export default function Integrations() {
+  return (
+    <DashboardCard
+      title="Integrations"
+      description="Connect your other social media accounts to Lumiere"
+    >
+      {SOCIAL_PLATFORMS.map((platform) => (
+        <div className="flex flex-col gap-4">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)